.dumi/changelog-plugin/hast-util-to-string.js

```
@@ -0,0 +1,54 @@
+ * @import {Nodes, Parents} from 'hast'
+ */
+
+/**
+ * Get the plain-text value of a hast node.
+ *
+ * @param {Nodes} node
+ *   Node to serialize.
+ * @returns {string}
+ *   Serialized node.
+ */
+export function toString(node) {
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)